tools/core-build-tasks/src/tasks/cleanCollateral.ts

```diff
@@ -5,14 +5,33 @@ import * as fs from 'fs';
 import * as Path from 'path';
 import rimraf from 'rimraf';
 import { getOrThrowFromProcess } from './helpers/getOrThrowFromProcess';
+import { getGameDeploymentRootPaths } from './helpers/getGameDeploymentRootPaths';
 
+/** @deprecated Use {@link getStandardCleanPaths} instead of {@link STANDARD_CLEAN_PATHS} */
 export const STANDARD_CLEAN_PATHS = [
     'LOCALAPPDATA/Packages/Microsoft.MinecraftUWP_8wekyb3d8bbwe/LocalState/games/com.mojang/development_behavior_packs/PROJECT_NAME',
     'LOCALAPPDATA/Packages/Microsoft.MinecraftUWP_8wekyb3d8bbwe/LocalState/games/com.mojang/development_resource_packs/PROJECT_NAME',
     'LOCALAPPDATA/Packages/Microsoft.MinecraftWindowsBeta_8wekyb3d8bbwe/LocalState/games/com.mojang/development_behavior_packs/PROJECT_NAME',
     'LOCALAPPDATA/Packages/Microsoft.MinecraftWindowsBeta_8wekyb3d8bbwe/LocalState/games/com.mojang/development_resource_packs/PROJECT_NAME',
 ];
 
+export function getStandardCleanPaths() {
+    const projectName = getOrThrowFromProcess('PROJECT_NAME');
+    const results = Object.values(getGameDeploymentRootPaths())
```

Comment:
Something to note here is that `getGameDeploymentRootPaths` doesn't use the `APPDATA` path, it only uses `LOCALAPPDATA` which would be a breaking change I suppose? I don't think the `copyTask` task would work if `LOCALAPPDATA` wasn't set correctly so I don't think this would actually be breaking anything.